### PR TITLE
#176 User Management: User List View

### PR DESCRIPTION
**Pull request by a2batic**
Bug-id: #176 
Signed-off-by: a2batic <murarkakanika@gmail.com>

### CODE REVIEW

**Comment by a2batic** [comment]
@anivargi @shtripat @nthomas-redhat @brainfunked @gnehapk @julienlim @cloudbehl , please review

File: specs/Spec_for_user_list_view.adoc [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+### UI
+
+There are following features involved in this view:-
+
+* Filter: To filter out User on the basis of:-
+** User ID (username)
+** Last Name
+** Role
+** Status
```
Comment (shtripat) [inline specs/Spec_for_user_list_view.adoc]: What are valid `status` values for user?

Comment (a2batic) [reply specs/Spec_for_user_list_view.adoc]: Its Enabled/Disabled, I have added it under Proposed workflow heading

File: specs/Spec_for_user_list_view.adoc [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+* email
+* username
+* name
+* role
+
+Required from API - 
+
+* status
+* notification
+
```
Comment (a2batic) [inline specs/Spec_for_user_list_view.adoc]: @anivargi @brainfunked @nthomas-redhat Are we supporting these features? If yes @anivargi please provide the updated response.

**Pull request merged**